libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/V2/Conway.hs

```diff
@@ -114,11 +114,11 @@ import Constrained qualified as C
 type ConwayUnivFns = StringFn : DefaultFns
 type ConwayFn = Fix (OneofL ConwayUnivFns)
 
-type IsConwayUniv fn =
+type ConwayUniverse fn =
```

Comment:
Can we have the changes to names, renaming, etc. in a separate PR. There are many things that need to be resolved about size and having a monster PR that touches everything in the codebase makes that very difficult.